Documentation/security/policy/language.rst

```diff
@@ -361,6 +361,12 @@ all
     The all entity represents the combination of all known clusters as well
     world and whitelists all communication.
 
+There are also optional ``fromNodes`` and ``toNodes`` fields that can be used to
```

Comment:
Instead of this paragraph I'd add a bullet to the list at the top (around L30). Does it make sense?

Reply:
yeah, sounds good, let me fix that